morello-pkglist

```diff
@@ -0,0 +1,45 @@
+#
+# This file contains the list of packages requested for use with the
+# Morello architecture on CheriBSD.  It is broken down into two sections, 
+# a set of packages known to build and the set of requeested packages
+# that do not build as yet.
+#
+# This file is meant to be used with the -f argument to poudriere bulk
+#
+# The current list of requested packages is maintained on 
+# https://github.com/CTSRD-CHERI/poudriere-infrastructure/wiki/Package-backlog
+#
+# Currently Working Packages 17 March 2022 <- Keep this current!
```

Comment:
I'd add an empty line to above this line to separate it from the main comment and make it better visible.